grails-app/views/consentGroup/show.gsp

```diff
@@ -29,9 +29,7 @@
         </span>
     </g:if>
     <div class="issue-type">
-      <span>New Status:</span>
-      <span>${issue?.approvalStatus}</span>
-      <br/><span>Information Sub-Status:</span>
+      <span>Information Sub-Status:</span>
```

Comment:
At first, I was going to suggest not capitalizing status in `Sub-Status`, but it turns out I would be wrong, it should be capitalized: https://www.businesswritingblog.com/business_writing/2010/08/capitalizing-hyphenated-words-in-titles-.html

Reply:
I really apreciate these links with info! thanks!